src/qibolab/platforms/abstract.py

```diff
@@ -16,6 +17,85 @@
 )
 
 
+class Qubit:
+    """Data structure that holds information about all instruments controlling a qubit.
+
+    Args:
+        index (int): Qubit index.
```

Comment:
Hi, AbstractPlatform supports both `int` and `str` types for qubit names. I know many were not aware of this and in some places we have default arguments like `qubit=0` that break that.
If possible I would suggest to continue supporting that functionality.
When characterising Boulos' qubits this was useful:
``` yaml
qubits: ['D001_250', 'D001_300_A', 'D001_300_B', 'D001_350']
```
It can also be useful in other chips to distinguish between regular and auxiliary qubits (witness qubits).